examples/2d_isometry/tiles.rs

```diff
@@ -0,0 +1,842 @@
+// TODO: Have something to import TextureAtlas as an asset.
+// This was generated by a simple parser built in 10 minutes,
+// but it's far from being convenient.
+
+use amethyst::assets::{AssetStorage, Loader};
+use amethyst::ecs::World;
+use amethyst::renderer::{
+    MaterialTextureSet, PngFormat, Sprite, SpriteSheet, SpriteSheetHandle, SpriteSheetSet, Texture,
+    TextureCoordinates, TextureMetadata, FilterMethod, SamplerInfo, WrapMode
+};
+
+const TEXTURE_ID: u64 = 0;
+const SPRITESHEET_SIZE: (f32, f32) = (1024.0, 2048.0);
+
+struct SpriteDescription {
```

Comment:
Isn't that redefining what prefab already is?

Reply:
I'm not aware of a prefab like this.

Reply:
Actually, you're not supposed to use prefabs for that sort of stuff.
A SpriteSheet is just an asset, it should not be in a prefab.